contrib/kea/files/kea-dhcp-ddns

```diff
@@ -0,0 +1,7 @@
+# kea-dhcp-ddns service
+
+type       = process
+command    = /usr/sbin/kea-dhcp-ddns -c /etc/kea/kea-dhcp-ddns.conf
+depends-on = network.target
+waits-for  = kea-dhcp4
```

Comment:
waits-for means that:  

- if kea-dhcp4 fails to start, it won't block kea-dhcp-ddns (which will still start)
- stopping kea-dhcp4 will not stop dhcp-ddns (it would be allowed)

is that correct?

Reply:
if you want the second, but not the first, then depends-ms is the same but does care about failure

otherwise if you want neither you want just a regular depends-on

Reply:
yes, waits-for just means "wait for the named service to succeed or fail" but does not imply any dependency

Reply:
there is also depends-ms, which like waits-for does not imply a dependency, but requires the named service to succeed starting

Reply:
I just want to decide the order services will start without creating any dependency so I'll replace it by `after` and I think it's enough